### DISCUSSION TITLE
Componente de conteúdo avançado D

### DISCUSSION THREAD

**Discussion by vinrosa** [Telefonica/mistica-design#1198] [Proposals]
### Goal

Criar um componente de conteúdo ”avançado” que será utilizado no componente data card através da propriedade de slot e que possa ser reutilizável de diversas maneiras e por combinações diferentes.

### Context

Os cards apresentados no novo conceito para o novo app Vivo possuem elementos de conteúdo que não são suportados pelo atual componente data card, a favor de suportar esse conteúdo, e com um racional de criar o menor número possível de variações, concluímos que com 4 novos componente de conteúdo  para uso da Vivo atenderemos os casos atuais da aba de assinaturas.
Esse componente suporta o componente de Compras

### Impact

Não gera impacto em componentes já existentes na biblioteca Mística.
Componente vai ser utilizado na lib de componentes avançados

### Discussion Questions

Alguma sugestão de melhoria?

### Additional considerations

Componentes relacionados:
https://github.com/Telefonica/mistica-design/discussions/1195
https://github.com/Telefonica/mistica-design/discussions/1196
https://github.com/Telefonica/mistica-design/discussions/1197

### Do you have a visual proposal?

![Comp D](https://user-images.githubusercontent.com/129896641/233357665-a7825b7a-836a-41cb-8f6c-676919955c02.jpg)

**Comment by yceballost** [comment]
Buenas @vinrosa!

Gracias de nuevo por la info 😃

Creemos que es posible que no sea necesario crear este slot para cubrir vuestro caso. Una vez esté disponible el stackingGroup debería de poderse cubrir esta necesidad sin crear nada más.

Aún así, esto puede cambiar una vez se analice en los PRDs los casos de uso y al final el equipo de CMS tendrá una respuesta más clara.

De momento, no lo consideraría un componente
